api-testing/multicloud-api/src/main/java/com/soda/utils/Constant.java

```diff
@@ -0,0 +1,23 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+/**
+ * In this class set Inputs path.
+ */
+public class Constant {
```

Comment:
Are we really using this class properly?
I mean all the Constants used in the project/module can be part of this class

Reply:
This is repeated qutions. Already discuss with @anvithks

Reply:
https://github.com/Click2cloud-Wadhai can you please put the reasons over here

Reply:
@himanshuvar Yes. All the constants will be use in the project from this class.